Sources/CLI/InitCommand.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import SwiftCLI
+import Configuration
+import Core
+import Git
+
+final class InitCommand: Command {
+	let name = "init"
+	let shortDescription = "Sets up Badonde for your project"
+	let longDescription =
+		"""
+		Sets up Badonde for your project.
+
+		When run inside a Git repository, this command will set up Badonde by:
+
+		  • Creating a config file at '.badonde/config.json'.
+		  • Prompting for GitHub and Jira credentials and adding them to the config.
+		  • Adding '.badonde' to gitignore.
+		  • Creating a basic Badondefile.
+
+		If either of these steps have previously ocurred, said step will be skipped.
+		"""
+
+	func execute() throws {
+		
```

Comment:
<!--
  3 failure:  Don't include ver..., Don't include ver..., Lines should not ...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/CLI/InitCommand.swift;
  Line: 25;
-->
- :no_entry_sign: Don't include vertical whitespace (empty line) after opening braces. (`vertical_whitespace_opening_braces`)
- :no_entry_sign: Don't include vertical whitespace (empty line) before closing braces. (`vertical_whitespace_closing_braces`)
- :no_entry_sign: Lines should not have trailing whitespace. (`trailing_whitespace`)